src/cli/serve.ts

```diff
@@ -17,10 +17,12 @@ program
     '--mapbox-access-token [mapboxAccessToken]',
     'Access Token for the Mapbox',
   )
+  .option('--headless', 'serve your map without opening a browser')
```

Comment:
Can we use the similar name of option with web pack?

Webpack has `--open` and `--no-open` option to avoid opening in browser. I think `headless` is not intuitive name for users.

https://webpack.js.org/configuration/dev-server/